src/python/ensembl/database/unittestdb.py

```diff
@@ -95,8 +99,14 @@ def __init__(self, url: URL, dump_dir: os.PathLike, name: Optional[str] = None)
                             conn.execute(f"TRUNCATE TABLE {table}")
                         else:
                             conn.execute(f"DELETE FROM {table}")
+
                         self._load_data(conn, table, filepath)
-        except:
+                if self.dbc.dialect == 'mysql':
+                    conn.execute(f"SET FOREIGN_KEY_CHECKS=1;")
+                elif self.dbc.dialect == 'sqlite':
+                    conn.execute(f"PRAGMA foreign_keys = ON;")
+        except Exception as exc:
+            print(exc)
```

Comment:
```suggestion
        except:
```
The exception is raised after dropping the database, so I believe there is no need to catch it and print it.

Reply:
Let's call this a typo :-D